src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -191,23 +190,7 @@ const convertTaskDetail = async (taskStatus, ports, userName, jobName, taskRoleN
   let affinityGroupName = null;
   // get container gpus
   let containerGpus = null;
-  try {
-    if (launcherConfig.enabledHived) {
-      const hivedSpec = yaml.load(pod.metadata.annotations['hivedscheduler.microsoft.com/pod-scheduling-spec']);
-      if (hivedSpec && hivedSpec.affinityGroup && hivedSpec.affinityGroup.name) {
-        affinityGroupName = hivedSpec.affinityGroup.name;
-      } else {
-        affinityGroupName = `default/${taskStatus.attemptStatus.podName}`;
```

Comment:
could you set it as default value for `affinityGroupName`?

Reply:
Done.